notebooks/work-in-progress/fuel-price-imputation/fuel-price-imputation-zane.ipynb

```diff
@@ -0,0 +1,1080 @@
+{
```

Comment:
<p>Oh interesting that you can only use sample weights with <code>cross_val_score</code>. You could use <code>search.fit(X_train, y_train)</code> where search is your cross validator and <code>hist_gbr__sample_weights</code> is part of your parameter grid </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/catalyst-cooperative/pudl/pull/1696/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>